lib/android-helpers.js

```diff
@@ -259,6 +266,10 @@ helpers.getLaunchInfo = async function (adb, opts) {
   }
   if (!appWaitActivity) {
     appWaitActivity = appActivity;
+  } else {
+    if (!new RegExp(/^[a-zA-Z0-9_\.]+$/).test(appWaitActivity)) {
```

Comment:
should appWaitPackage have this verification?

Reply:
Yeah, since users can set value in appWaitpackage by themselves.